lib/src/edhoc.rs

```diff
@@ -1737,8 +1737,8 @@ mod tests {
         let c_i_tv = C_I_TV;
 
         // the actual value will be zeroed since it doesn't matter in this test
-        let mut ead_value = EdhocMessageBuffer::new();
-        ead_value.len = MAX_MESSAGE_SIZE_LEN;
+        let mut ead_value = EdhocBuffer::new();
+        ead_value.len = MAX_EAD_LEN;
```

Comment:
That I almost missed that is a strong indication that we should not have this pub in the first place; that's a panic bound to happen, especially when a user runs things on different size parametrizations than any hax panic-free proof runs on.